### PR TITLE
feat: adds GH raw source URL

### PR DESCRIPTION
**Pull request by nilekhc**
Signed-off-by: Nilekh Chaudhari <1626598+nilekhc@users.noreply.github.com>

fixes: https://github.com/open-policy-agent/gatekeeper-library/issues/233

Changes are available on my fork website: https://nilekhc.github.io/gatekeeper-library/

### CODE REVIEW

**Comment by nilekhc** [comment]
cc @maxsmythe @sozercan @ritazh

File: scripts/website/template.md [hunk 1]
```diff
@@ -13,5 +13,9 @@ title: %TITLE%
 %TEMPLATE%
 ```
 
+Usage
```
Comment (sozercan) [inline scripts/website/template.md:16]:
```suggestion
### Usage
```

**Review by sozercan — APPROVED** [review]
small nit, otherwise LGTM. Thank you!

**Review by maxsmythe — APPROVED** [review]
LGTM

**Comment by nilekhc** [comment]
@sozercan @maxsmythe I resolved the comments. PTAL. Thanks!

**Pull request merged**